public/utils/constants.ts

```diff
@@ -53,6 +53,8 @@ export const ANOMALY_RESULT_INDEX = '.opendistro-anomaly-results';
 
 export const BASE_DOCS_LINK = 'https://opensearch.org/docs/monitoring-plugins';
 
+export const AD_DOCS_LINK = 'https://opensearch.org/docs/latest/monitoring-plugins/anomaly-detection/index/';
```

Comment:
This link actually was broken for me when testing, I had to use `${BASE_DOCS_LINK}/ad`. LMK if this either works for you or if not then you should try to what worked on my end.